var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -55,29 +56,37 @@ class Adios(AutotoolsPackage):
     variant('zlib', default=True, description='Enable zlib transform support')
     variant('bzip2', default=False, description='Enable bzip2 transform support')
     variant('szip', default=False, description='Enable szip transform support')
-    variant('zfp', default=False, description='Enable ZFP transform support')
+    variant('zfp', default=True, description='Enable ZFP transform support')
+    variant('sz', default=False, description='Enable SZ transform support')
     # transports and serial file converters
     variant('hdf5', default=False, description='Enable parallel HDF5 transport and serial bp2h5 converter')
-
-    # Lots of setting up here for this package
-    # module swap PrgEnv-intel PrgEnv-$COMP
-    # module load cray-hdf5/1.8.14
-    # module load python/2.7.10
+    variant('netcdf', default=False, description='Enable netcdf support')
+    variant('flexpath', default=False, description='Enable flexpath transport')
+    variant('dataspaces', default=False, description='Enable dataspaces transport')
+    variant('staging', default=False, description='Enable dataspaces and flexpath staging transports')
 
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
-    depends_on('libtool', type='build')
+    depends_on('libtool@:2.4.2', type='build')
     depends_on('python', type='build')
 
     depends_on('mpi', when='+mpi')
-    depends_on('mxml@2.9:')
     # optional transformations
     depends_on('zlib', when='+zlib')
     depends_on('bzip2', when='+bzip2')
     depends_on('szip', when='+szip')
+    depends_on('sz@develop', when='+sz')
```

Comment:
that is true, but that means until sz builds @kshitij-v-mehta needs to tell people to build via `spack install somethingOnSz ^sz@develop`.

(e.g. savanna: `spack install savanna ^adios+sz@develop`)